src/iblt.cpp

```diff
@@ -117,6 +138,13 @@ void CIblt::resize(size_t _expectedNumEntries)
 
     CIblt::n_hash = OptimalNHash(_expectedNumEntries);
 
+    if (salt > VALS_32 / n_hash)
+        throw std::runtime_error("salt * n_hash must fit in uint32_t");
+
+    // set hash seeds from salt
+    for (size_t i = 0; i < n_hash; i++)
+        mapHashIdxSeeds[i] = salt * n_hash + i;
```

Comment:
biggest number is salt * n_hash + n_hash - 1.  Replacing salt with the check above -> (VALS_32 / n_hash * n_hash) + n_hash -1, or VALS_32 + n_hash - 1.  So your check is too lenient by n_hash.

Reply:
Good point, but this line has been changed in 4eee6a1.